website/content/docs/configuration/controller.mdx

```diff
@@ -108,6 +108,33 @@ description will be read.
   are anything specified by Go's [ParseDuration()](https://golang.org/pkg/time/#ParseDuration) method. Only
   used when an `ops` listener is set and the Controller is present. Default is 0 seconds.
 
+- `api_rate_limit` - Sets limits on the rate of requests for controller API endpoints.
```

Comment:
We should add `api_rate_limit_max_quotas` and `api_rate_limit_disable` to the controller stanza.